### ISSUE TITLE
UIE finetune evaluate.py 报错

### ISSUE THREAD

**Issue by minlik** [PaddlePaddle/PaddleNLP#2413]
```
/usr/local/lib/python3.7/dist-packages/scipy/fft/__init__.py:97: DeprecationWarning: The module numpy.dual is deprecated.  Instead of using dual, use the functions directly from numpy or scipy.
  from numpy.dual import register_func
/usr/local/lib/python3.7/dist-packages/scipy/sparse/sputils.py:17: DeprecationWarning: `np.typeDict` is a deprecated alias for `np.sctypeDict`.
  supported_dtypes = [np.typeDict[x] for x in supported_dtypes]
/usr/local/lib/python3.7/dist-packages/flatbuffers/compat.py:19: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
/usr/local/lib/python3.7/dist-packages/scipy/special/orthogonal.py:81: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  from numpy import (exp, inf, pi, sqrt, floor, sin, cos, around, int,
[2022-06-05 16:50:00,946] [    INFO] - We are using <class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'> to load './checkpoint/model_best/'.
Traceback (most recent call last):
  File "evaluate.py", line 90, in <module>
    do_eval()
  File "evaluate.py", line 55, in do_eval
    tokenizer = AutoTokenizer.from_pretrained(args.model_path)
  File "/usr/local/lib/python3.7/dist-packages/paddlenlp/transformers/auto/tokenizer.py", line 234, in from_pretrained
    pretrained_model_name_or_path, *model_args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/paddlenlp/transformers/tokenizer_utils_base.py", line 1623, in from_pretrained
    '') and os.path.isfile(file_path):
TypeError: isfile() takes 1 positional argument but 2 were given
```
1）PaddleNLP和PaddlePaddle版本：PaddleNLP 2.3.1，PaddlePaddle 2.3.0
2）系统环境：Python 3.7.13

**Comment by zouhan6806504** [comment]
> ```
> /usr/local/lib/python3.7/dist-packages/scipy/fft/__init__.py:97: DeprecationWarning: The module numpy.dual is deprecated.  Instead of using dual, use the functions directly from numpy or scipy.
>   from numpy.dual import register_func
> /usr/local/lib/python3.7/dist-packages/scipy/sparse/sputils.py:17: DeprecationWarning: `np.typeDict` is a deprecated alias for `np.sctypeDict`.
>   supported_dtypes = [np.typeDict[x] for x in supported_dtypes]
> /usr/local/lib/python3.7/dist-packages/flatbuffers/compat.py:19: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
>   import imp
> /usr/local/lib/python3.7/dist-packages/scipy/special/orthogonal.py:81: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
> Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
>   from numpy import (exp, inf, pi, sqrt, floor, sin, cos, around, int,
> [2022-06-05 16:50:00,946] [    INFO] - We are using <class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'> to load './checkpoint/model_best/'.
> Traceback (most recent call last):
>   File "evaluate.py", line 90, in <module>
>     do_eval()
>   File "evaluate.py", line 55, in do_eval
>     tokenizer = AutoTokenizer.from_pretrained(args.model_path)
>   File "/usr/local/lib/python3.7/dist-packages/paddlenlp/transformers/auto/tokenizer.py", line 234, in from_pretrained
>     pretrained_model_name_or_path, *model_args, **kwargs)
>   File "/usr/local/lib/python3.7/dist-packages/paddlenlp/transformers/tokenizer_utils_base.py", line 1623, in from_pretrained
>     '') and os.path.isfile(file_path):
> TypeError: isfile() takes 1 positional argument but 2 were given
> ```
> 
> 1）PaddleNLP和PaddlePaddle版本：PaddleNLP 2.3.1，PaddlePaddle 2.3.0 2）系统环境：Python 3.7.13

我也是这样，前两天还正常，今天就出错了。估计是版本升级导致的

**Comment by laoduo** [comment]
俺也一样，遇到了同样的问题。
Python:3.9.12
paddle:2.3.0
paddlenlp:2.3.2

[[2022-06-06 06:33:19,429] [    INFO] - We are using <class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'> to load './checkpoint/model_best'.
Traceback (most recent call last):
  File "/opt/soft/PaddleNLP-develop/model_zoo/uie/evaluate.py", line 90, in <module>
    do_eval()
  File "/opt/soft/PaddleNLP-develop/model_zoo/uie/evaluate.py", line 55, in do_eval
    tokenizer = AutoTokenizer.from_pretrained(args.model_path)
  File "/root/anaconda3/envs/paddle/lib/python3.9/site-packages/paddlenlp/transformers/auto/tokenizer.py", line 233, in from_pretrained
    return tokenizer_class.from_pretrained(
  File "/root/anaconda3/envs/paddle/lib/python3.9/site-packages/paddlenlp/transformers/tokenizer_utils_base.py", line 1622, in from_pretrained
    elif not os.path.isfile(init_kwargs[args_name],
TypeError: isfile() takes 1 positional argument but 2 were given

**Comment by guoshengCS** [comment]
这里有点bug，已经提交PR修复了，也会今天重新发个fix版本。可以先在报错这里去掉`''`这个先行使用，抱歉带来困扰
<img width="786" alt="image" src="https://user-images.githubusercontent.com/14105589/172275702-4bdfc975-c37d-423c-9690-cfa9f7d3a34f.png">

**Comment by guoshengCS** [comment]
Resolved by #2424

**Comment by ZeyuChen** [comment]
This issues will resolved by #2424 and release paddlenlp 2.3.3
